app/controllers/catalog_admin/containers_controller.rb

```diff
@@ -73,7 +73,7 @@ def container_params
     params.require(:container).permit(
       :slug,
       :row_order_position,
-      :lcoale,
+      :locale,
```

Comment:
I don't think this line is still needed. The `locale` param is now called `l` and is retrieved directly from the `params` hash.

Reply:
Alright, done !